### PR TITLE
Minor updates to the OMR guide

### PR DESCRIPTION
**Pull request by stevsmit**
Preview: https://deploy-preview-43351--osdocs.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-creating-registry

Jiras: 

https://issues.redhat.com/browse/PROJQUAY-3383
https://issues.redhat.com/browse/PROJQUAY-3388
https://issues.redhat.com/browse/PROJQUAY-3387
https://issues.redhat.com/browse/PROJQUAY-3385

QE approved.

### CODE REVIEW

File: modules/mirror-registry-flags.adoc [hunk 1]
```diff
@@ -12,6 +12,7 @@ The following flags are available for the _mirror registry for Red Hat OpenShift
 | Flags | Description
 | `--autoApprove` | A boolean value that disables interactive prompts. If set to `true`, the `quayRoot` directory is automatically deleted when uninstalling the mirror registry. Defaults to `false` if left unspecified.
 | `--initPassword` | The password of the init user created during Quay installation. Must be at least eight characters and contain no whitespace.
+|`--initUser string` | Shows the password of the initial user. Defaults to `init` if left unspecified.
```
Comment (dongboyan77) [inline modules/mirror-registry-flags.adoc:15]: Should be `The username of the initial user`, the original description is wrong in help page, DEV will fix it in next release

File: installing/disconnected_install/installing-mirroring-creating-registry.adoc [hunk 2]
```diff
@@ -21,6 +21,7 @@ If you already have a container image registry, such as Red Hat Quay, you can sk
 * 2 or more vCPUs.
 * 8 GB of RAM.
 * About 12 GB for {product-title} {product-version} release images, or about 358 GB for {product-title} {product-version} release images and {product-title} {product-version} Red Hat Operator images. Up to 1 TB per stream or more is suggested.
+* An OpenSSL certificate for your registry.
```
Comment (dongboyan77) [inline installing/disconnected_install/installing-mirroring-creating-registry.adoc:24]: I think the available updates should be installing openssl package like podman, reference to

**Comment by dongboyan77** [comment]
Except above two issues, others look good to me

**Review by HammerMeetNail — APPROVED** [review]
LGTM 👍

File: installing/disconnected_install/installing-mirroring-creating-registry.adoc [hunk 3]
```diff
@@ -14,7 +14,7 @@ If you already have a container image registry, such as Red Hat Quay, you can sk
 == Prerequisites
 
 * An {product-title} subscription.
-* {op-system-base-full} 8 with Podman 3.3 installed.
+* {op-system-base-full} 8 with Podman 3.3 and openssl installed.
```
Comment (mburke5678) [inline installing/disconnected_install/installing-mirroring-creating-registry.adoc:17]: It seems that we use `openssl` vs `OpenSSL` to an alarming degree. However, I think the [latter is correct](https://www.openssl.org/).

File: modules/mirror-registry-update.adoc [hunk 4]
```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-installation-images.adoc
+
+[id="upgrading-mirror-registry_{context}"]
+= Upgrading the mirror registry for Red Hat OpenShift
```
Comment (mburke5678) [inline modules/mirror-registry-update.adoc:6]: Is there a reason you didn't use {product-title}?

**Comment by mburke5678** [comment]
@stevsmit A couple of minor nits. Otherwise LGTM

File: modules/mirror-registry-update.adoc (hunk 4 above)
Comment (stevsmit) [reply modules/mirror-registry-update.adoc:6]: @mburke5678 yeah, the name of the tool itself is "mirror registry for Red Hat OpenShift." It's a terrible name, but just kinda what I'm stuck with!

**Pull request merged**